platforms/documentation/docs/src/docs/userguide/reference/best-practices/best_practices_general.adoc

```diff
@@ -142,6 +142,143 @@ include::sample[dir="snippets/bestPractices/usePluginsBlock-do/groovy",files="bu
 
 `<<tags_reference.adoc#tag:structuring-builds,#structuring-builds>>`
 
+[[dont_assume_plugin_order]]
+== Don’t Assume your Plugin is Applied after Another
+
+Gradle does not guarantee the order in which plugins are applied, especially across multiple build scripts, projects, convention plugins, or included builds.
```

Comment:
Gradle does guarantee order, but that order might not be obvious from looking at a single place.